src/CSAccounting.sol

```diff
@@ -33,6 +33,8 @@ contract CSAccounting is
     bytes32 public constant RESUME_ROLE = keccak256("RESUME_ROLE");
     bytes32 public constant MANAGE_BOND_CURVES_ROLE =
         keccak256("MANAGE_BOND_CURVES_ROLE");
+
+    /// @dev Deprecated. Use setBondCurve with the corresponding role on the CSM contract instead
```

Comment:
Can we remove it? This is not a storage, hence this is no longer needed

Reply:
Still relevant

Reply:
changed the code to not use it but forgot to remove, thx